MediaAdmin/FileUtils/Image/ImagickImageManager.php

```diff
@@ -45,23 +42,26 @@ public function generateAlternative($filePath, array $format)
      *
      * @param array   $format
      * @param Imagick $image
-     * 
+     *
      * @return Imagick
      */
     protected function resizeImage(array $format, Imagick $image)
     {
-        $maxWidth = array_key_exists('max_width', $format)? $format['max_width']: -1;
-        $maxHeight = array_key_exists('max_height', $format)? $format['max_height']: -1;
+        $maxWidth = array_key_exists('max_width', $format) ? $format['max_width']: -1;
+        $maxHeight = array_key_exists('max_height', $format) ? $format['max_height']: -1;
 
         if (-2 != $maxWidth + $maxHeight) {
             $image->setimagebackgroundcolor('#000000');
             $refRatio = $maxWidth / $maxHeight;
-            $imageRatio = $image->getImageWidth() / $image->getImageHeight();
-
-            if ($refRatio > $imageRatio || $maxWidth == -1) {
-                $image = $this->resizeOnHeight($image, $maxHeight);
-            } else {
-                $image = $this->resizeOnWidth($image, $maxWidth);
+            $imageRatioByWidth = $image->getImageHeight() / $image->getImageWidth();
+            $imageRatioByHeight = $image->getImageWidth() / $image->getImageHeight();
```

Comment:
I don't understand the functionnality of this 2 variables. Initially, the code compares the uploded image ratio (for instance 4:3) to the alternative ratio (for instance 16:9) to determine if the image must be redim on width or height. But I don't see the logic with these 2 new variables.